worlds/rabi_ribi/__init__.py

```diff
@@ -29,6 +32,26 @@ def launch_client():
     component_type=Type.CLIENT
 ))
 
+class RabiRibiSettings(settings.Group):
+    class GameInstallationPath(settings.UserFolderPath):
+        """
+        The installation folder of the game.
+        """
+        description = "Rabi-Ribi Installation Path"
+
+    game_installation_path: GameInstallationPath = GameInstallationPath("C:/Program Files (x86)/Steam/steamapps/common/Rabi-Ribi")
+
+class RabiRibiWeb(WebWorld):
+    """Web integration for Rabi-Ribi"""
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up Rabi-Ribi integration for Archipelago multiworld games.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["Phie", "PsyMarth"]
+    )]
```

Comment:
I'm a fan of leaving every module to do one thing. I would encourage the settings and the web to be in its own dedicated module. I think this is also the convention for most APWorlds within the repo.

Reply:
interesting... ig I was mistaken haha. ack 👍